src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services10.php

```diff
@@ -51,6 +51,9 @@ protected function getTestService()
     public function getParameter(string $name): array|bool|string|int|float|\UnitEnum|null
     {
         if (!(isset($this->parameters[$name]) || isset($this->loadedDynamicParameters[$name]) || \array_key_exists($name, $this->parameters))) {
+            if ('.' === $name[0] && isset($this->getRemovedParameters()[$name])) {
```

Comment:
I've now removed that `getRemovedParameters()` method.
Should we dump the array (now passed as option) anyway for the consistency of this check? Or should we only check for a starting dot?